<!DOCTYPE html>
<html lang="en" prefix="og: http://ogp.me/ns# fb: https://www.facebook.com/2008/fbml">
<head>
    <title>Nathan A. Baker</title>
    <!-- Using the latest rendering mode for IE -->
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">


    <link href="./images/favicon.ico" rel="icon">

<link rel="canonical" href=".">
        <meta name="author" content="Nathan Baker" />

    <!-- Open Graph tags -->
        <meta property="og:site_name" content="Nathan A. Baker" />
        <meta property="og:type" content="website"/>
        <meta property="og:title" content="Nathan A. Baker"/>
        <meta property="og:url" content="."/>
        <meta property="og:description" content="Nathan A. Baker"/>


    <!-- Bootstrap -->
        <link rel="stylesheet" href="./theme/css/bootstrap.min.css" type="text/css"/>
    <link href="./theme/css/font-awesome.min.css" rel="stylesheet">

    <link href="./theme/css/pygments/native.css" rel="stylesheet">
    <link rel="stylesheet" href="./theme/css/style.css" type="text/css"/>





</head>
<body>

<div class="navbar navbar-default navbar-fixed-top" role="navigation">
	<div class="container">
        <div class="navbar-header">
            <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-ex1-collapse">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
            </button>
            <a href="./" class="navbar-brand">
Nathan A. Baker            </a>
        </div>
        <div class="collapse navbar-collapse navbar-ex1-collapse">
            <ul class="nav navbar-nav">
                         <li><a href="./pages/contact.html">
                             Contact
                          </a></li>
                        <li >
                            <a href="./category/publications.html">Publications</a>
                        </li>
                        <li >
                            <a href="./category/research.html">Research</a>
                        </li>
            </ul>
            <ul class="nav navbar-nav navbar-right">
              <li><a href="./archives.html"><i class="fa fa-th-list"></i><span class="icon-label">Archives</span></a></li>
            </ul>
        </div>
        <!-- /.navbar-collapse -->
    </div>
</div> <!-- /.navbar -->
<!-- Banner -->
<!-- End Banner -->
<div class="container">
    <div class="row">
        <div class="col-sm-9">
            <article>
                <h2><a href="./dna-dna-interactions-with-atomic-detail.html">DNA-DNA interactions with atomic detail</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2012-09-28T09:01:00-07:00"> Fri 28 September 2012</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/electrostatics.html">electrostatics</a>
        /
	<a href="./tag/dna.html">dna</a>
        /
	<a href="./tag/current_research.html">current_research</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>This project explores the fundamental physics that governs interactions between nucleic acids in aqueous electrolyte solutions.</p>
                    <a class="btn btn-default btn-xs" href="./dna-dna-interactions-with-atomic-detail.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./collaboratory-on-mathematics-for-mesoscopic-modeling-of-materials-cm4.html">Collaboratory on Mathematics for Mesoscopic Modeling of Materials (CM4)</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2012-08-01T09:01:00-07:00"> Wed 01 August 2012</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/materials.html">materials</a>
        /
	<a href="./tag/mathematics.html">mathematics</a>
        /
	<a href="./tag/multiscale.html">multiscale</a>
        /
	<a href="./tag/uq.html">uq</a>
        /
	<a href="./tag/current_research.html">current_research</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>CM4 focuses on developing rigorous mathematical foundations for understanding and controlling fundamental mechanisms in mesoscale processes to enable scalable synthesis of complex materials, through the design of efficient modeling methods and corresponding scalable algorithms.</p>
                    <a class="btn btn-default btn-xs" href="./collaboratory-on-mathematics-for-mesoscopic-modeling-of-materials-cm4.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./us-eu-nanotechnology-communities-of-research.html">US-EU Nanotechnology Communities of Research</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2012-01-01T09:01:00-08:00"> Sun 01 January 2012</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/nanotechnology.html">nanotechnology</a>
        /
	<a href="./tag/ontology.html">ontology</a>
        /
	<a href="./tag/informatics.html">informatics</a>
        /
	<a href="./tag/working_group.html">working_group</a>
        /
	<a href="./tag/past_research.html">past_research</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>The Databases and Ontologies US-EU CoR was created to enable international coordination on nanoinformatics research.</p>
                    <a class="btn btn-default btn-xs" href="./us-eu-nanotechnology-communities-of-research.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./mechanism-of-oxysterol-activation-of-membrane-cholesterol.html">Mechanism of Oxysterol Activation of Membrane Cholesterol</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2011-01-01T09:01:00-08:00"> Sat 01 January 2011</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/biomembranes.html">biomembranes</a>
        /
	<a href="./tag/cholesterol.html">cholesterol</a>
        /
	<a href="./tag/oxysterols.html">oxysterols</a>
        /
	<a href="./tag/past_research.html">past_research</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>This project used computational biophysical methods to explore the mechanisms by which oxysterols alter biomembrane behavior.</p>
                    <a class="btn btn-default btn-xs" href="./mechanism-of-oxysterol-activation-of-membrane-cholesterol.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./signature-discovery-initiative.html">Signature Discovery Initiative</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2010-10-01T09:01:00-07:00"> Fri 01 October 2010</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/signatures.html">signatures</a>
        /
	<a href="./tag/mathematics.html">mathematics</a>
        /
	<a href="./tag/statistics.html">statistics</a>
        /
	<a href="./tag/machine_learning.html">machine_learning</a>
        /
	<a href="./tag/current_research.html">current_research</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>The Signature Discovery Initiative is a 6-year internal investment by Pacific Northwest National Laboratory to transform the process by which signatures are discovered and used.</p>
                    <a class="btn btn-default btn-xs" href="./signature-discovery-initiative.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./characterizationbioinformatics-modeling-of-nanoparticle-complement-interactions.html">Characterization/bioinformatics-modeling of nanoparticle-complement interactions</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2010-09-27T09:01:00-07:00"> Mon 27 September 2010</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/nanotechnology.html">nanotechnology</a>
        /
	<a href="./tag/ontology.html">ontology</a>
        /
	<a href="./tag/informatics.html">informatics</a>
        /
	<a href="./tag/past_research.html">past_research</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>This project developed statistical models to predict blood toxicity of nanoemulsion diagnostic and therapeutic nanoparticles.</p>
                    <a class="btn btn-default btn-xs" href="./characterizationbioinformatics-modeling-of-nanoparticle-complement-interactions.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./isa-tab-nano.html">ISA-TAB-Nano</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2010-01-01T09:01:00-08:00"> Fri 01 January 2010</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/nanotechnology.html">nanotechnology</a>
        /
	<a href="./tag/ontology.html">ontology</a>
        /
	<a href="./tag/software.html">software</a>
        /
	<a href="./tag/informatics.html">informatics</a>
        /
	<a href="./tag/past_research.html">past_research</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>ISA-TAB-Nano was developed to help share and analyze data from cancer nanotechnology research.</p>
                    <a class="btn btn-default btn-xs" href="./isa-tab-nano.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./geometric-flow-approach-to-implicit-solvation-modeling.html">Geometric flow approach to implicit solvation modeling</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2009-08-01T09:01:00-07:00"> Sat 01 August 2009</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/electrostatics.html">electrostatics</a>
        /
	<a href="./tag/solvation.html">solvation</a>
        /
	<a href="./tag/software.html">software</a>
        /
	<a href="./tag/mathematics.html">mathematics</a>
        /
	<a href="./tag/geometric_flow.html">geometric_flow</a>
        /
	<a href="./tag/past_research.html">past_research</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>This project developed new robust models of biomolecular solvation that coupled polar and nonpolar influences.</p>
                    <a class="btn btn-default btn-xs" href="./geometric-flow-approach-to-implicit-solvation-modeling.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./a-multiscale-model-linking-ion-channel-molecular-dynamics-and-electrostatics-to-the-cardiac-action-potential.html">A multiscale model linking ion-channel molecular dynamics and electrostatics to the cardiac action potential</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2009-01-01T00:00:00-08:00"> Thu 01 January 2009</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/ion_channels.html">ion_channels</a>
        /
	<a href="./tag/multiscale.html">multiscale</a>
        /
	<a href="./tag/electrostatics.html">electrostatics</a>
        /
	<a href="./tag/molecular_dynamics.html">molecular_dynamics</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>Ion-channel function is determined by its gating movement. Yet, molecular dynamics and electrophysiological simulations were never combined to link molecular structure to function. We performed multiscale molecular dynamics and continuum electrostatics calculations to simulate a cardiac K(+) channel (I(Ks)) gating and its alteration by mutations that cause arrhythmias and sudden death. An all-atom model of the I(Ks) alpha-subunit KCNQ1, based on the recent Kv1.2 structure, is used to calculate electrostatic energies during gating. Simulations are compared with experiments where varying degrees of positive charge-added via point mutation-progressively reduce current. Whole-cell simulations show that mutations cause action potential and ECG QT interval prolongation, consistent with clinical phenotypes. This framework allows integration of multiscale observations to study the molecular basis of excitation and its alteration by disease.</p>
                    <a class="btn btn-default btn-xs" href="./a-multiscale-model-linking-ion-channel-molecular-dynamics-and-electrostatics-to-the-cardiac-action-potential.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./cancer-nanotechnology-knowledgebase-for-nanoparticle-analysis-and-design.html">Cancer Nanotechnology Knowledgebase for Nanoparticle Analysis and Design</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2008-11-01T09:01:00-07:00"> Sat 01 November 2008</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/nanotechnology.html">nanotechnology</a>
        /
	<a href="./tag/ontology.html">ontology</a>
        /
	<a href="./tag/informatics.html">informatics</a>
        /
	<a href="./tag/software.html">software</a>
        /
	<a href="./tag/past_research.html">past_research</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>This project developed knowledgebase informatics resources for sharing and analyzing nanotechnology data for cancer diagnosis and treatment.</p>
                    <a class="btn btn-default btn-xs" href="./cancer-nanotechnology-knowledgebase-for-nanoparticle-analysis-and-design.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./ncip-nanotechnology-working-group.html">NCIP Nanotechnology Working Group</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2008-01-01T09:01:00-08:00"> Tue 01 January 2008</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/nanotechnology.html">nanotechnology</a>
        /
	<a href="./tag/ontology.html">ontology</a>
        /
	<a href="./tag/informatics.html">informatics</a>
        /
	<a href="./tag/working_group.html">working_group</a>
        /
	<a href="./tag/past_research.html">past_research</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>The National Cancer Institute (NCI) National Cancer Informatics Program (NCIP) Nanotechnology Working Group was established for researchers with a specific interest in informatics and computational approaches to nanotechnology, with a particular emphasis on nanomedicine.</p>
                    <a class="btn btn-default btn-xs" href="./ncip-nanotechnology-working-group.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./molecular-dynamics-simulation-of-the-escherichia-coli-nikr-protein-equilibrium-conformational-fluctuations-reveal-interdomain-allosteric-communication-pathways.html">Molecular dynamics simulation of the Escherichia coli NikR protein: equilibrium conformational fluctuations reveal interdomain allosteric communication pathways</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2008-01-01T00:00:00-08:00"> Tue 01 January 2008</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/allostery.html">allostery</a>
        /
	<a href="./tag/nikr.html">nikr</a>
        /
	<a href="./tag/molecular_dynamics.html">molecular_dynamics</a>
        /
	<a href="./tag/metalloregulation.html">metalloregulation</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>Escherichia coli NikR is a homotetrameric Ni 2+- and DNA-binding protein that functions as a transcriptional repressor of the NikABCDE nickel permease. The protein is composed of two distinct domains. The N-terminal 50 amino acids of each chain forms part of the dimeric ribbon-helix-helix (RHH) domains, a well-studied DNA-binding fold. The 83-residue C-terminal nickel-binding domain forms an ACT (aspartokinase, chorismate mutase, and TyrA) fold and contains the tetrameric interface. In this study, we have utilized an equilibrium molecular dynamics simulation in order to explore the conformational dynamics of the NikR tetramer and determine important residue interactions within and between the RHH and ACT domains to gain insight into the effects of Ni 2+ on DNA-binding activity. The molecular simulation data were analyzed using two different correlation measures based on fluctuations in atomic position and noncovalent contacts together with a clustering algorithm to define groups of residues with similar correlation patterns for both types of correlation measure. Based on these analyses, we have defined a series of residue interrelationships that describe an allosteric communication pathway between the Ni 2+- and DNA-binding sites, which are separated by 40????. Several of the residues identified by our analyses have been previously shown experimentally to be important for NikR function. An additional subset of the identified residues structurally connects the experimentally implicated residues and may help coordinate the allosteric communication between the ACT and RHH domains. ?? 2008 Elsevier Ltd. All rights reserved.</p>
                    <a class="btn btn-default btn-xs" href="./molecular-dynamics-simulation-of-the-escherichia-coli-nikr-protein-equilibrium-conformational-fluctuations-reveal-interdomain-allosteric-communication-pathways.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./molecular-dynamics-simulations-of-asymmetric-nacl-and-kcl-solutions-separated-by-phosphatidylcholine-bilayers-potential-drops-and-structural-changes-induced-by-strong-na-lipid-interactions-and-finite-size-effects.html">Molecular dynamics simulations of asymmetric NaCl and KCl solutions separated by phosphatidylcholine bilayers: potential drops and structural changes induced by strong Na+-lipid interactions and finite size effects</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2008-01-01T00:00:00-08:00"> Tue 01 January 2008</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/electrostatics.html">electrostatics</a>
        /
	<a href="./tag/molecular_dynamics.html">molecular_dynamics</a>
        /
	<a href="./tag/biomembranes.html">biomembranes</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>Differences of ionic concentrations across lipid bilayers are some of the primary energetic driving forces for cellular electrophysiology. While macroscopic models of asymmetric ionic solutions are well-developed, their connection to ion, water, and lipid interactions at the atomic scale are much more poorly understood. In this study, we used molecular dynamics to examine a system of two chambers of equal ionic strength, but differing amounts of NaCl and KCl, separated by a lipid bilayer. Our expectation was that the net electrostatic potential difference between the two chambers should be small or zero. Contrary to our expectation, a large potential difference (-70 mV) slowly evolved across the two water chambers over the course of our 172-ns simulation. This potential primarily originated from strong Na(+) binding to the carbonyls of the phosphatidylcholine lipids. This ion adsorption also led to significant structural and mechanical changes in the lipid bilayer. We discuss this surprising result in the context of indirect experimental evidence for Na(+) interaction with bilayers as well as potential caveats in current biomembrane simulation methodology, including force-field parameters and finite size effects.</p>
                    <a class="btn btn-default btn-xs" href="./molecular-dynamics-simulations-of-asymmetric-nacl-and-kcl-solutions-separated-by-phosphatidylcholine-bilayers-potential-drops-and-structural-changes-induced-by-strong-na-lipid-interactions-and-finite-size-effects.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./finite-element-analysis-of-the-time-dependent-smoluchowski-equation-for-acetylcholinesterase-reaction-rate-calculations.html">Finite element analysis of the time-dependent Smoluchowski equation for acetylcholinesterase reaction rate calculations</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2007-01-01T00:00:00-08:00"> Mon 01 January 2007</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/diffusion.html">diffusion</a>
        /
	<a href="./tag/acetylcholinesterase.html">acetylcholinesterase</a>
        /
	<a href="./tag/finite_elements.html">finite_elements</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>This article describes the numerical solution of the time-dependent Smoluchowski equation to study diffusion in biomolecular systems. Specifically, finite element methods have been developed to calculate ligand binding rate constants for large biomolecules. The resulting software has been validated and applied to the mouse acetylcholinesterase (mAChE) monomer and several tetramers. Rates for inhibitor binding to mAChE were calculated at various ionic strengths with several different time steps. Calculated rates show very good agreement with experimental and theoretical steady-state studies. Furthermore, these finite element methods require significantly fewer computational resources than existing particle-based Brownian dynamics methods and are robust for complicated geometries. The key finding of biological importance is that the rate accelerations of the monomeric and tetrameric mAChE that result from electrostatic steering are preserved under the non-steady-state conditions that are expected to occur in physiological circumstances.</p>
                    <a class="btn btn-default btn-xs" href="./finite-element-analysis-of-the-time-dependent-smoluchowski-equation-for-acetylcholinesterase-reaction-rate-calculations.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./optimizing-the-poisson-dielectric-boundary-with-explicit-solvent-forces-and-energies-lessons-learned-with-atom-centered-dielectric-functions.html">Optimizing the Poisson dielectric boundary with explicit solvent forces and energies: lessons learned with atom-centered dielectric functions</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2007-01-01T00:00:00-08:00"> Mon 01 January 2007</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/electrostatics.html">electrostatics</a>
        /
	<a href="./tag/molecular_dynamics.html">molecular_dynamics</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>Accurate implicit solvent models require parameters that have been optimized in a system- or atom-specific manner on the basis of experimental data or more rigorous explicit solvent simulations. Models based on the Poisson or Poisson?Boltzmann equation are particularly sensitive to the nature and location of the boundary which separates the low dielectric solute from the high dielectric solvent. Here, we present a novel method for optimizing the solute radii, which define the dielectric boundary, on the basis of forces and energies from explicit solvent simulations. We use this method to optimize radii for protein systems defined by AMBER ff99 partial charges and a spline-smoothed solute surface. The spline-smoothed surface is an atom-centered dielectric function that enables stable and efficient force calculations. We explore the relative performance of radii optimized with forces alone and those optimized with forces and energies. We show that our radii reproduce the explicit solvent forces and energies more accurately than four other parameter sets commonly used in conjunction with the AMBER force field, each of which has been appropriately scaled for spline-smoothed surfaces. Finally, we demonstrate that spline-smoothed surfaces show surprising accuracy for small, compact systems but may have limitations for highly solvated protein systems. The optimization method presented here is efficient and applicable to any system with explicit solvent parameters. It can be used to determine the optimal continuum parameters when experimental solvation energies are unavailable and the computational costs of explicit solvent charging free energies are prohibitive. Accurate implicit solvent models require parameters that have been optimized in a system- or atom-specific manner on the basis of experimental data or more rigorous explicit solvent simulations. Models based on the Poisson or Poisson-Boltzmann equation are particularly sensitive to the nature and location of the boundary which separates the low dielectric solute from the high dielectric solvent. Here, we present a novel method for optimizing the solute radii, which define the dielectric boundary, on the basis of forces and energies from explicit solvent simulations. We use this method to optimize radii for protein systems defined by AMBER ff99 partial charges and a spline-smoothed solute surface. The spline-smoothed surface is an atom-centered dielectric function that enables stable and efficient force calculations. We explore the relative performance of radii optimized with forces alone and those optimized with forces and energies. We show that our radii reproduce the explicit solvent forces and energies more accurately than four other parameter sets commonly used in conjunction with the AMBER force field, each of which has been appropriately scaled for spline-smoothed surfaces. Finally, we demonstrate that spline-smoothed surfaces show surprising accuracy for small, compact systems but may have limitations for highly solvated protein systems. The optimization method presented here is efficient and applicable to any system with explicit solvent parameters. It can be used to determine the optimal continuum parameters when experimental solvation energies are unavailable and the computational costs of explicit solvent charging free energies are prohibitive.</p>
                    <a class="btn btn-default btn-xs" href="./optimizing-the-poisson-dielectric-boundary-with-explicit-solvent-forces-and-energies-lessons-learned-with-atom-centered-dielectric-functions.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./pdb2pqr-expanding-and-upgrading-automated-preparation-of-biomolecular-structures-for-molecular-simulations.html">PDB2PQR: expanding and upgrading automated preparation of biomolecular structures for molecular simulations</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2007-01-01T00:00:00-08:00"> Mon 01 January 2007</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/solvation.html">solvation</a>
        /
	<a href="./tag/software.html">software</a>
        /
	<a href="./tag/electrostatics.html">electrostatics</a>
        /
	<a href="./tag/titration.html">titration</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>Real-world observable physical and chemical characteristics are increasingly being calculated from the 3D structures of biomolecules. Methods for calculating pK(a) values, binding constants of ligands, and changes in protein stability are readily available, but often the limiting step in computational biology is the conversion of PDB structures into formats ready for use with biomolecular simulation software. The continued sophistication and integration of biomolecular simulation methods for systems- and genome-wide studies requires a fast, robust, physically realistic and standardized protocol for preparing macromolecular structures for biophysical algorithms. As described previously, the PDB2PQR web server addresses this need for electrostatic field calculations (Dolinsky et al., Nucleic Acids Research, 32, W665-W667, 2004). Here we report the significantly expanded PDB2PQR that includes the following features: robust standalone command line support, improved pK(a) estimation via the PROPKA framework, ligand parameterization via PEOE_PB charge methodology, expanded set of force fields and easily incorporated user-defined parameters via XML input files, and improvement of atom addition and optimization code. These features are available through a new web interface (http://pdb2pqr.sourceforge.net/), which offers users a wide range of options for PDB file conversion, modification and parameterization.</p>
                    <a class="btn btn-default btn-xs" href="./pdb2pqr-expanding-and-upgrading-automated-preparation-of-biomolecular-structures-for-molecular-simulations.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./polarizable-atomic-multipole-solutes-in-a-poisson-boltzmann-continuum.html">Polarizable atomic multipole solutes in a Poisson-Boltzmann continuum</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2007-01-01T00:00:00-08:00"> Mon 01 January 2007</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/force_fields.html">force_fields</a>
        /
	<a href="./tag/solvation.html">solvation</a>
        /
	<a href="./tag/electrostatics.html">electrostatics</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>Modeling the change in the electrostatics of organic molecules upon moving from vacuum into solvent, due to polarization, has long been an interesting problem. In vacuum, experimental values for the dipole moments and polarizabilities of small, rigid molecules are known to high accuracy; however, it has generally been difficult to determine these quantities for a polar molecule in water. A theoretical approach introduced by Onsager [J. Am. Chem. Soc. 58, 1486 (1936)] used vacuum properties of small molecules, including polarizability, dipole moment, and size, to predict experimentally known permittivities of neat liquids via the Poisson equation. Since this important advance in understanding the condensed phase, a large number of computational methods have been developed to study solutes embedded in a continuum via numerical solutions to the Poisson-Boltzmann equation. Only recently have the classical force fields used for studying biomolecules begun to include explicit polarization in their functional forms. Here the authors describe the theory underlying a newly developed polarizable multipole Poisson-Boltzmann (PMPB) continuum electrostatics model, which builds on the atomic multipole optimized energetics for biomolecular applications (AMOEBA) force field. As an application of the PMPB methodology, results are presented for several small folded proteins studied by molecular dynamics in explicit water as well as embedded in the PMPB continuum. The dipole moment of each protein increased on average by a factor of 1.27 in explicit AMOEBA water and 1.26 in continuum solvent. The essentially identical electrostatic response in both models suggests that PMPB electrostatics offers an efficient alternative to sampling explicit solvent molecules for a variety of interesting applications, including binding energies, conformational analysis, and pK(a) prediction. Introduction of 150 mM salt lowered the electrostatic solvation energy between 2 and 13 kcalmole, depending on the formal charge of the protein, but had only a small influence on dipole moments.</p>
                    <a class="btn btn-default btn-xs" href="./polarizable-atomic-multipole-solutes-in-a-poisson-boltzmann-continuum.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./solvent-reaction-field-potential-inside-an-uncharged-globular-protein-a-bridge-between-implicit-and-explicit-solvent-models.html">Solvent reaction field potential inside an uncharged globular protein: A bridge between implicit and explicit solvent models?</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2007-01-01T00:00:00-08:00"> Mon 01 January 2007</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/solvation.html">solvation</a>
        /
	<a href="./tag/electrostatics.html">electrostatics</a>
        /
	<a href="./tag/molecular_dynamics.html">molecular_dynamics</a>
        /
	<a href="./tag/nonpolar.html">nonpolar</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>The solvent reaction field potential of an uncharged protein immersed in simple point charge/extended explicit solvent was computed over a series of molecular dynamics trajectories, in total 1560 ns of simulation time. A finite, positive potential of 13-24 kbTec(-1) (where T=300 K), dependent on the geometry of the solvent-accessible surface, was observed inside the biomolecule. The primary contribution to this potential arose from a layer of positive charge density 1.0 A from the solute surface, on average 0.008 ec/A3, which we found to be the product of a highly ordered first solvation shell. Significant second solvation shell effects, including additional layers of charge density and a slight decrease in the short-range solvent-solvent interaction strength, were also observed. The impact of these findings on implicit solvent models was assessed by running similar explicit solvent simulations on the fully charged protein system. When the energy due to the solvent reaction field in the uncharged system is accounted for, correlation between per-atom electrostatic energies for the explicit solvent model and a simple implicit (Poisson) calculation is 0.97, and correlation between per-atom energies for the explicit solvent model and a previously published, optimized Poisson model is 0.99.</p>
                    <a class="btn btn-default btn-xs" href="./solvent-reaction-field-potential-inside-an-uncharged-globular-protein-a-bridge-between-implicit-and-explicit-solvent-models.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./allosteric-regulation-of-the-nickel-dependent-nikr-repressor.html">Allosteric Regulation of the Nickel-dependent NikR Repressor</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2006-02-01T09:01:00-08:00"> Wed 01 February 2006</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/nikr.html">nikr</a>
        /
	<a href="./tag/metalloregulation.html">metalloregulation</a>
        /
	<a href="./tag/past_research.html">past_research</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>This project studied the allosteric mechanisms that govern metalloregulation by the bacterial protein NikR.</p>
                    <a class="btn btn-default btn-xs" href="./allosteric-regulation-of-the-nickel-dependent-nikr-repressor.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./assessing-implicit-models-for-nonpolar-mean-solvation-forces-the-importance-of-dispersion-and-volume-terms.html">Assessing implicit models for nonpolar mean solvation forces: the importance of dispersion and volume terms</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2006-01-01T00:00:00-08:00"> Sun 01 January 2006</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/solvation.html">solvation</a>
        /
	<a href="./tag/nonpolar.html">nonpolar</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>Continuum solvation models provide appealing alternatives to explicit solvent methods because of their ability to reproduce solvation effects while alleviating the need for expensive sampling. Our previous work has demonstrated that Poisson-Boltzmann methods are capable of faithfully reproducing polar explicit solvent forces for dilute protein systems; however, the popular solvent-accessible surface area model was shown to be incapable of accurately describing nonpolar solvation forces at atomic-length scales. Therefore, alternate continuum methods are needed to reproduce nonpolar interactions at the atomic scale. In the present work, we address this issue by supplementing the solvent-accessible surface area model with additional volume and dispersion integral terms suggested by scaled particle models and Weeks-Chandler-Andersen theory, respectively. This more complete nonpolar implicit solvent model shows very good agreement with explicit solvent results and suggests that, although often overlooked, the inclusion of appropriate dispersion and volume terms are essential for an accurate implicit solvent description of atomic-scale nonpolar forces.</p>
                    <a class="btn btn-default btn-xs" href="./assessing-implicit-models-for-nonpolar-mean-solvation-forces-the-importance-of-dispersion-and-volume-terms.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./electrostatic-properties-of-cowpea-chlorotic-mottle-virus-and-cucumber-mosaic-virus-capsids.html">Electrostatic properties of cowpea chlorotic mottle virus and cucumber mosaic virus capsids</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2006-01-01T00:00:00-08:00"> Sun 01 January 2006</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/electrostatics.html">electrostatics</a>
        /
	<a href="./tag/finite_differences.html">finite_differences</a>
        /
	<a href="./tag/finite_elements.html">finite_elements</a>
        /
	<a href="./tag/virus.html">virus</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>Electrostatic properties of cowpea chlorotic mottle virus (CCMV) and cucumber mosaic virus (CMV) were investigated using numerical solutions to the Poisson-Boltzmann equation. Experimentally, it has been shown that CCMV particles swell in the absence of divalent cations when the pH is raised from 5 to 7. CMV, although structurally homologous, does not undergo this transition. An analysis of the calculated electrostatic potential confirms that a strong electrostatic repulsion at the calcium-binding sites in the CCMV capsid is most likely the driving force for the capsid swelling process during the release of calcium. The binding interaction between the encapsulated genome material (RNA) inside of the capsid and the inner capsid shell is weakened during the swelling transition. This probably aids in the RNA release process, but it is unlikely that the RNA is released through capsid openings due to unfavorable electrostatic interaction between the RNA and capsid inner shell residues at these openings. Calculations of the calcium binding energies show that Ca(2+) can bind both to the native and swollen forms of the CCMV virion. Favorable binding to the swollen form suggests that Ca(2+) ions can induce the capsid contraction and stabilize the native form.</p>
                    <a class="btn btn-default btn-xs" href="./electrostatic-properties-of-cowpea-chlorotic-mottle-virus-and-cucumber-mosaic-virus-capsids.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./nanoparticle-ontology-npo.html">NanoParticle Ontology (NPO)</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2005-09-30T09:01:00-07:00"> Fri 30 September 2005</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/nanotechnology.html">nanotechnology</a>
        /
	<a href="./tag/ontology.html">ontology</a>
        /
	<a href="./tag/software.html">software</a>
        /
	<a href="./tag/informatics.html">informatics</a>
        /
	<a href="./tag/past_research.html">past_research</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>The NanoParticle Ontology was developed to help share and analyze data from cancer nanotechnology research.</p>
                    <a class="btn btn-default btn-xs" href="./nanoparticle-ontology-npo.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./the-siteman-cancer-center-nanotechnology-excellence-at-washington-university.html">The Siteman Cancer Center Nanotechnology Excellence at Washington University</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2005-09-01T09:01:00-07:00"> Thu 01 September 2005</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/nanotechnology.html">nanotechnology</a>
        /
	<a href="./tag/ontology.html">ontology</a>
        /
	<a href="./tag/informatics.html">informatics</a>
        /
	<a href="./tag/past_research.html">past_research</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>This project developed informatics resources for sharing and analyzing nanotechnology data for cancer diagnosis and treatment.</p>
                    <a class="btn btn-default btn-xs" href="./the-siteman-cancer-center-nanotechnology-excellence-at-washington-university.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./improving-implicit-solvent-simulations-a-poisson-centric-view.html">Improving implicit solvent simulations: a Poisson-centric view</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2005-01-01T00:00:00-08:00"> Sat 01 January 2005</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/solvation.html">solvation</a>
        /
	<a href="./tag/review.html">review</a>
        /
	<a href="./tag/electrostatics.html">electrostatics</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>Recent developments in implicit solvent models may be compared in terms of accuracy and computational efficiency. Based on improvements in the accuracy of generalized Born methods and the speed of Poisson-Boltzmann solvers, it appears that the two techniques are converging to a point at which both will be suitable for simulating certain types of biomolecular systems over sizable time and length scales.</p>
                    <a class="btn btn-default btn-xs" href="./improving-implicit-solvent-simulations-a-poisson-centric-view.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./iron-responsive-element-rna-flexibility-described-by-nmr-and-isotropic-reorientational-eigenmode-dynamics.html">Iron responsive element RNA flexibility described by NMR and isotropic reorientational eigenmode dynamics</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2005-01-01T00:00:00-08:00"> Sat 01 January 2005</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/rna.html">rna</a>
        /
	<a href="./tag/molecular_dynamics.html">molecular_dynamics</a>
        /
	<a href="./tag/nucleic_acids.html">nucleic_acids</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>The first example of the application of reorientational eigenmode dynamics (RED) to RNA is shown here for the small and floppy Iron Responsive Element (IRE) RNA hairpin. Order parameters calculated for bases and riboses from a 12 ns molecular dynamics trajectory are compared to experimentally determined order parameters from 13C-1H NMR relaxation experiments, and shown to be in qualitative agreement. Given the small size of the IRE hairpin and its very flexible loop, isotropic RED (iRED) was also used to analyze the trajectory in order to describe its dynamic motions. iRED analysis shows that the global and internal dynamics of the IRE are not rigorously separable, which will result in inaccurate experimental order parameters. In addition, the iRED analysis described the many correlated motions that comprise the dynamics of the IRE RNA. The combined use of NMR relaxation, RED, and iRED provide a uniquely detailed description of IRE RNA dynamics.</p>
                    <a class="btn btn-default btn-xs" href="./iron-responsive-element-rna-flexibility-described-by-nmr-and-isotropic-reorientational-eigenmode-dynamics.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./molecular-dynamics-simulations-of-salicylate-effects-on-the-micro-and-mesoscopic-properties-of-a-dipalmitoylphosphatidylcholine-bilayer.html">Molecular dynamics simulations of salicylate effects on the micro-and mesoscopic properties of a dipalmitoylphosphatidylcholine bilayer</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2005-01-01T00:00:00-08:00"> Sat 01 January 2005</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/molecular_dynamics.html">molecular_dynamics</a>
        /
	<a href="./tag/biomembranes.html">biomembranes</a>
        /
	<a href="./tag/salicylate.html">salicylate</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>Salicylate, an amphiphilic molecule and a popular member of the nonsteroidal anti-inflammatory drug family, is known to affect hearing through reduction of the electromechanical coupling in the outer hair cells of the ear. This reduction of electromotility by salicylate has been widely studied, but the molecular mechanism of the phenomenon is still unknown. In this study, we investigated one aspect of salicylate's action, namely the perturbation of electrical and mechanical membrane properties by salicylate in the absence of cytoskeletal or membrane-bound motor proteins such as prestin. In particular, we simulated the interaction of salicylate with a dipalmitoylphosphatidylcholine (DPPC) bilayer via atomically detailed molecular dynamics simulations to observe the effect of salicylate on the microscopic and mesoscopic properties of the bilayer. The results demonstrate that salicylate interacts with the bilayer by associating at the water-DPPC interface in a nearly perpendicular orientation and penetrating more deeply into the bilayer than either sodium or chloride. This association has several affects on the membrane properties. First, binding of salicylate to the membrane displaces chloride from the bilayer-water interface. Second, salicylate influences the electrostatic potential and dielectric properties of the bilayer, with significant changes at the water-lipid bilayer interface. Third, salicylate association results in structural changes, including decreased headgroup area per lipid and increased lipid tail order. However, salicylate does not significantly alter the mechanical properties of the DPPC bilayer; bulk compressibility, area compressibility, and bending modulus were only perturbed by small, statistically insignificant amounts by the presence of salicylate. The observations from these simulations are in qualitative agreement with experimental data and support the conclusion that salicylate influences the electrical but not the mechanical properties of DPPC membranes.</p>
                    <a class="btn btn-default btn-xs" href="./molecular-dynamics-simulations-of-salicylate-effects-on-the-micro-and-mesoscopic-properties-of-a-dipalmitoylphosphatidylcholine-bilayer.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./tetrameric-mouse-acetylcholinesterase-continuum-diffusion-rate-calculations-by-solving-the-steady-state-smoluchowski-equation-using-finite-element-methods.html">Tetrameric mouse acetylcholinesterase: continuum diffusion rate calculations by solving the steady-state Smoluchowski equation using finite element methods</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2005-01-01T00:00:00-08:00"> Sat 01 January 2005</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/diffusion.html">diffusion</a>
        /
	<a href="./tag/electrostatics.html">electrostatics</a>
        /
	<a href="./tag/acetylcholinesterase.html">acetylcholinesterase</a>
        /
	<a href="./tag/finite_elements.html">finite_elements</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>The tetramer is the most important form for acetylcholinesterase in physiological conditions, i.e., in the neuromuscular junction and the nervous system. It is important to study the diffusion of acetylcholine to the active sites of the tetrameric enzyme to understand the overall signal transduction process in these cellular components. Crystallographic studies revealed two different forms of tetramers, suggesting a flexible tetramer model for acetylcholinesterase. Using a recently developed finite element solver for the steady-state Smoluchowski equation, we have calculated the reaction rate for three mouse acetylcholinesterase tetramers using these two crystal structures and an intermediate structure as templates. Our results show that the reaction rates differ for different individual active sites in the compact tetramer crystal structure, and the rates are similar for different individual active sites in the other crystal structure and the intermediate structure. In the limit of zero salt, the reaction rates per active site for the tetramers are the same as that for the monomer, whereas at higher ionic strength, the rates per active site for the tetramers are approximately 67-75 \% of the rate for the monomer. By analyzing the effect of electrostatic forces on ACh diffusion, we find that electrostatic forces play an even more important role for the tetramers than for the monomer. This study also shows that the finite element solver is well suited for solving the diffusion problem within complicated geometries.</p>
                    <a class="btn btn-default btn-xs" href="./tetrameric-mouse-acetylcholinesterase-continuum-diffusion-rate-calculations-by-solving-the-steady-state-smoluchowski-equation-using-finite-element-methods.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./pdb2pqr-an-automated-pipeline-for-the-setup-of-poisson-boltzmann-electrostatics-calculations.html">PDB2PQR: an automated pipeline for the setup of Poisson-Boltzmann electrostatics calculations</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2004-07-01T00:00:00-07:00"> Thu 01 July 2004</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/solvation.html">solvation</a>
        /
	<a href="./tag/software.html">software</a>
        /
	<a href="./tag/electrostatics.html">electrostatics</a>
        /
	<a href="./tag/titration.html">titration</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>Continuum solvation models, such as Poisson-Boltzmann and Generalized Born methods, have become increasingly popular tools for investigating the influence of electrostatics on biomolecular structure, energetics and dynamics. However, the use of such methods requires accurate and complete structural data as well as force field parameters such as atomic charges and radii. Unfortunately, the limiting step in continuum electrostatics calculations is often the addition of missing atomic coordinates to molecular structures from the Protein Data Bank and the assignment of parameters to biomolecular structures. To address this problem, we have developed the PDB2PQR web service (http://agave.wustl.edu/pdb2pqr/). This server automates many of the common tasks of preparing structures for continuum electrostatics calculations, including adding a limited number of missing heavy atoms to biomolecular structures, estimating titration states and protonating biomolecules in a manner consistent with favorable hydrogen bonding, assigning charge and radius parameters from a variety of force fields, and finally generating 'PQR' output compatible with several popular computational biology packages. This service is intended to facilitate the setup and execution of electrostatics calculations for both experts and non-experts and thereby broaden the accessibility to the biological community of continuum electrostatics analyses of biomolecular systems.</p>
                    <a class="btn btn-default btn-xs" href="./pdb2pqr-an-automated-pipeline-for-the-setup-of-poisson-boltzmann-electrostatics-calculations.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./national-biomedical-computation-resource.html">National Biomedical Computation Resource</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2004-06-01T09:01:00-07:00"> Tue 01 June 2004</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/electrostatics.html">electrostatics</a>
        /
	<a href="./tag/solvation.html">solvation</a>
        /
	<a href="./tag/software.html">software</a>
        /
	<a href="./tag/titration.html">titration</a>
        /
	<a href="./tag/past_research.html">past_research</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>NBCR supported early development of the PDB2PQR software and continues to host the web server.</p>
                    <a class="btn btn-default btn-xs" href="./national-biomedical-computation-resource.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./apbs-and-pdb2pqr.html">APBS and PDB2PQR</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2004-01-15T00:00:00-08:00"> Thu 15 January 2004</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/electrostatics.html">electrostatics</a>
        /
	<a href="./tag/solvation.html">solvation</a>
        /
	<a href="./tag/software.html">software</a>
        /
	<a href="./tag/titration.html">titration</a>
        /
	<a href="./tag/mathematics.html">mathematics</a>
        /
	<a href="./tag/current_research.html">current_research</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>APBS and PDB2PQR are software packages designed to assist in the understanding of biomolecular solvation and electrostatics.</p>
                    <a class="btn btn-default btn-xs" href="./apbs-and-pdb2pqr.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./continuum-diffusion-reaction-rate-calculations-of-wild-type-and-mutant-mouse-acetylcholinesterase-adaptive-finite-element-analysis.html">Continuum diffusion reaction rate calculations of wild-type and mutant mouse acetylcholinesterase: adaptive finite element analysis</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2004-01-01T00:00:00-08:00"> Thu 01 January 2004</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/diffusion.html">diffusion</a>
        /
	<a href="./tag/acetylcholinesterase.html">acetylcholinesterase</a>
        /
	<a href="./tag/finite_elements.html">finite_elements</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>As described previously, continuum models, such as the Smoluchowski equation, offer a scalable framework for studying diffusion in biomolecular systems. This work presents new developments in the efficient solution of the continuum diffusion equation. Specifically, we present methods for adaptively refining finite element solutions of the Smoluchowski equation based on a posteriori error estimates. We also describe new, molecular-surface-based models, for diffusional reaction boundary criteria and compare results obtained from these models with the traditional spherical criteria. The new methods are validated by comparison of the calculated reaction rates with experimental values for wild-type and mutant forms of mouse acetylcholinesterase. The results show good agreement with experiment and help to define optimal reactive boundary conditions.</p>
                    <a class="btn btn-default btn-xs" href="./continuum-diffusion-reaction-rate-calculations-of-wild-type-and-mutant-mouse-acetylcholinesterase-adaptive-finite-element-analysis.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./electrostatic-interaction-between-rna-and-protein-capsid-in-cowpea-chlorotic-mottle-virus-simulated-by-a-coarse-grain-rna-model-and-a-monte-carlo-approach.html">Electrostatic interaction between RNA and protein capsid in cowpea chlorotic mottle virus simulated by a coarse-grain RNA model and a Monte Carlo approach</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2004-01-01T00:00:00-08:00"> Thu 01 January 2004</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/virus.html">virus</a>
        /
	<a href="./tag/dna.html">dna</a>
        /
	<a href="./tag/rna.html">rna</a>
        /
	<a href="./tag/coarse-graining.html">coarse-graining</a>
        /
	<a href="./tag/monte_carlo.html">monte_carlo</a>
        /
	<a href="./tag/electrostatics.html">electrostatics</a>
        /
	<a href="./tag/nucleic_acids.html">nucleic_acids</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>Although many viruses have been crystallized and the protein capsid structures have been determined by x-ray crystallography, the nucleic acids often cannot be resolved. This is especially true for RNA viruses. The lack of information about the conformation of DNA/RNA greatly hinders our understanding of the assembly mechanism of various viruses. Here we combine a coarse-grain model and a Monte Carlo method to simulate the distribution of viral RNA inside the capsid of cowpea chlorotic mottle virus. Our results show that there is very strong interaction between the N-terminal residues of the capsid proteins, which are highly positive charged, and the viral RNA. Without these residues, the binding energy disfavors the binding of RNA by the capsid. The RNA forms a shell close to the capsid with the highest densities associated with the capsid dimers. These high-density regions are connected to each other in the shape of a continuous net of triangles. The overall icosahedral shape of the net overlaps with the capsid subunit icosahedral organization. Medium density of RNA is found under the pentamers of the capsid. These findings are consistent with experimental observations.</p>
                    <a class="btn btn-default btn-xs" href="./electrostatic-interaction-between-rna-and-protein-capsid-in-cowpea-chlorotic-mottle-virus-simulated-by-a-coarse-grain-rna-model-and-a-monte-carlo-approach.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./finite-element-solution-of-the-steady-state-smoluchowski-equation-for-rate-constant-calculations.html">Finite element solution of the steady-state Smoluchowski equation for rate constant calculations</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2004-01-01T00:00:00-08:00"> Thu 01 January 2004</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/diffusion.html">diffusion</a>
        /
	<a href="./tag/finite_elements.html">finite_elements</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>This article describes the development and implementation of algorithms to study diffusion in biomolecular systems using continuum mechanics equations. Specifically, finite element methods have been developed to solve the steady-state Smoluchowski equation to calculate ligand binding rate constants for large biomolecules. The resulting software has been validated and applied to mouse acetylcholinesterase. Rates for inhibitor binding to mAChE were calculated at various ionic strengths with several different reaction criteria. The calculated rates were compared with experimental data and show very good agreement when the correct reaction criterion is used. Additionally, these finite element methods require significantly less computational resources than existing particle-based Brownian dynamics methods.</p>
                    <a class="btn btn-default btn-xs" href="./finite-element-solution-of-the-steady-state-smoluchowski-equation-for-rate-constant-calculations.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./isim-a-program-for-grand-canonical-monte-carlo-simulations-of-the-ionic-environment-of-biomolecules.html">ISIM: a program for grand canonical Monte Carlo simulations of the ionic environment of biomolecules</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2004-01-01T00:00:00-08:00"> Thu 01 January 2004</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/electrostatics.html">electrostatics</a>
        /
	<a href="./tag/software.html">software</a>
        /
	<a href="./tag/monte_carlo.html">monte_carlo</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>In this work we present a new software package (ISIM), which represents a flexible, computational tool for simulations of electrolyte solutions via a grand canonical Monte Carlo procedure (GCMC) with a specific capability of treating biomolecules in solution. The GCMC method provides a powerful tool for studying the ionic environments of highly charged macromolecules with attention to the atomic detail of both the solute and the mobile counterions. The ISIM software differs from previous schemes mainly by treating different ion types independently and offering a new parameterization procedure for calibrating excess chemical potentials and bulk ion concentrations. Additionally, ISIM leverages the APBS software package to provide accurate descriptions of the biomolecular electrostatic potential through the efficient solution of Poisson's equation. ISIM has been validated on a variety of test systems; we successfully reproduce elementary properties of electrolyte solutions as well as theoretical and experime...</p>
                    <a class="btn btn-default btn-xs" href="./isim-a-program-for-grand-canonical-monte-carlo-simulations-of-the-ionic-environment-of-biomolecules.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./solvation-forces-on-biomolecular-structures-a-comparison-of-explicit-solvent-and-poisson-boltzmann-models.html">Solvation forces on biomolecular structures: a comparison of explicit solvent and Poisson-Boltzmann models</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2004-01-01T00:00:00-08:00"> Thu 01 January 2004</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/solvation.html">solvation</a>
        /
	<a href="./tag/electrostatics.html">electrostatics</a>
        /
	<a href="./tag/molecular_dynamics.html">molecular_dynamics</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>Continuum electrostatics methods have become increasingly popular due to their ability to provide approximate descriptions of solvation energies and forces without expensive sampling required by explicit solvent models. In particular, the Poisson-Boltzmann equation (PBE) provides electrostatic potentials, solvation energies, and forces by modeling the solvent as a featureless, dielectric material, and the mobile ions as a continuous distribution of charge. Polar solvation forces and energies obtained from the PBE are often supplemented with simple solvent-accessible surface area (SASA) models of nonpolar solvation. Given the recent development of methods that enable the use of PBE and SASA forces in molecular dynamics simulations, it is important to determine the ability of these implicit solvent models to accurately reproduce the solvation forces of more detailed explicit solvent simulations. In this article, we compare PBE and SASA solvation forces with explicit solvent forces for several snapshots from eight trajectories of static conformations of intestinal fatty acid binding protein. The results from this comparison show that current implementations of the PBE are capable of generating polar solvation forces that correlate well with explicit solvent forces but systematically overestimate the magnitude of the interaction. However, SASA-based nonpolar forces are found to have no significant correlation with nonpolar explicit solvent forces. Nevertheless, due to the small magnitude of the nonpolar forces in the current system, a good correlation is still obtained for total solvation forces. The good correlation of implicit solvent forces with more detailed explicit solvent models is encouraging and implies that the systematic errors identified in these models could be corrected by appropriate parameterization of the force fields.</p>
                    <a class="btn btn-default btn-xs" href="./solvation-forces-on-biomolecular-structures-a-comparison-of-explicit-solvent-and-poisson-boltzmann-models.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./finite-element-simulations-of-acetylcholine-diffusion-in-neuromuscular-junctions.html">Finite element simulations of acetylcholine diffusion in neuromuscular junctions</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2003-01-01T00:00:00-08:00"> Wed 01 January 2003</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/diffusion.html">diffusion</a>
        /
	<a href="./tag/acetylcholinesterase.html">acetylcholinesterase</a>
        /
	<a href="./tag/finite_elements.html">finite_elements</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>A robust infrastructure for solving time-dependent diffusion using the finite element package FEtk has been developed to simulate synaptic transmission in a neuromuscular junction with realistic postsynaptic folds. Simplified rectilinear synapse models serve as benchmarks in initial numerical studies of how variations in geometry and kinetics relate to endplate currents associated with fast-twitch, slow-twitch, and dystrophic muscles. The flexibility and scalability of FEtk affords increasingly realistic and complex models that can be formed in concert with expanding experimental understanding from electron microscopy. Ultimately, such models may provide useful insight on the functional implications of controlled changes in processes, suggesting therapies for neuromuscular diseases.</p>
                    <a class="btn btn-default btn-xs" href="./finite-element-simulations-of-acetylcholine-diffusion-in-neuromuscular-junctions.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./poisson-boltzmann-methods-for-biomolecular-electrostatics.html">Poisson-Boltzmann methods for biomolecular electrostatics</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2003-01-01T00:00:00-08:00"> Wed 01 January 2003</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/solvation.html">solvation</a>
        /
	<a href="./tag/review.html">review</a>
        /
	<a href="./tag/electrostatics.html">electrostatics</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>This chapter presents Poisson-Boltzmann (PB) methods for biomolecular electrostatics. The understanding of electrostatic properties is a basic aspect of the investigation of biomolecular processes. Structures of proteins and other biopolymers are being determined at an increasing rate through structural genomics and other effort. The nonlinear equations obtained from the full PB equation require more specialized techniques, such as Newton methods, to determine the solution to the discretized algebraic equation. The increasing size and number of biomolecular structures have necessitated the development of new parallel finite difference and finite element methods for solving the PB equation. These parallel techniques allow users to leverage supercomputing resources to determine the electrostatic properties of single-structure calculations on large biological systems consisting of millions of atoms. In addition, advances in improving the efficiency of PB force calculations have paved the way for a new era of biomolecular dynamics simulation methods using detailed implicit solvent models. It is anticipated that PB methods will continue to play an important role in computational biology as the study of biological systems grow from macromolecular to cellular scales.</p>
                    <a class="btn btn-default btn-xs" href="./poisson-boltzmann-methods-for-biomolecular-electrostatics.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./the-physical-basis-of-microtubule-structure-and-stability.html">The physical basis of microtubule structure and stability</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2003-01-01T00:00:00-08:00"> Wed 01 January 2003</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/solvation.html">solvation</a>
        /
	<a href="./tag/electrostatics.html">electrostatics</a>
        /
	<a href="./tag/microtubules.html">microtubules</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>Microtubules are cylindrical polymers found in every eukaryotic cell. They have a unique helical structure that has implications at both the cellular level, in terms of the functions they perform, and at the multicellular level, such as determining the left-right symmetry in plants. Through the combination of an atomically detailed model for a microtubule and large-scale computational techniques for computing electrostatic interactions, we are able to explain the observed microtubule structure. On the basis of the lateral interactions between protofilaments, we have determined that B lattice is the most favorable configuration. Further, we find that these lateral bonds are significantly weaker than the longitudinal bonds along protofilaments. This explains observations of microtubule disassembly and may serve as another step toward understanding the basis for dynamic instability.</p>
                    <a class="btn btn-default btn-xs" href="./the-physical-basis-of-microtubule-structure-and-stability.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./binding-of-aminoglycoside-antibiotics-to-the-small-ribosomal-subunit-a-continuum-electrostatics-investigation.html">Binding of aminoglycoside antibiotics to the small ribosomal subunit: a continuum electrostatics investigation</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2002-01-01T00:00:00-08:00"> Tue 01 January 2002</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/solvation.html">solvation</a>
        /
	<a href="./tag/electrostatics.html">electrostatics</a>
        /
	<a href="./tag/ribosome.html">ribosome</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>The binding of paromomycin and similar antibiotics to the small (30S) ribosomal subunit has been studied using continuum electrostatics methods. Crystallographic information from a complex of paromomycin with the 30S subunit was used as a framework to develop structures of similar antibiotics in the same ribosomal binding site. Total binding energies were calculated from electrostatic properties obtained by solution of the Poisson−Boltzmann equation combined with a surface area-dependent apolar term. These computed results showed good correlation with experimental data. Additionally, calculation of the ribosomal electrostatic potential in the paromomycin binding site provided insight into the electrostatic mechanisms for aminoglycoside binding and clues for the rational design of more effective antibiotics.</p>
                    <a class="btn btn-default btn-xs" href="./binding-of-aminoglycoside-antibiotics-to-the-small-ribosomal-subunit-a-continuum-electrostatics-investigation.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./bridging-implicit-and-explicit-solvent-approaches-for-membrane-electrostatics.html">Bridging implicit and explicit solvent approaches for membrane electrostatics</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2002-01-01T00:00:00-08:00"> Tue 01 January 2002</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/solvation.html">solvation</a>
        /
	<a href="./tag/electrostatics.html">electrostatics</a>
        /
	<a href="./tag/biomembranes.html">biomembranes</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>Conformations of a zwitterionic bilayer were sampled from a molecular dynamics simulation and their electrostatic properties analyzed by solution of the Poisson equation. These traditionally implicit electrostatic calculations were performed in the presence of varying amounts of explicit solvent to assess the magnitude of error introduced by a uniform dielectric description of water surrounding the bilayer. It was observed that membrane dipole potential calculations in the presence of explicit water were significantly different than wholly implicit solvent calculations with the calculated dipole potential converging to a reasonable value when four or more hydration layers were included explicitly.</p>
                    <a class="btn btn-default btn-xs" href="./bridging-implicit-and-explicit-solvent-approaches-for-membrane-electrostatics.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./electrostatics-of-nanosystems-application-to-microtubules-and-the-ribosome.html">Electrostatics of nanosystems: application to microtubules and the ribosome</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2001-08-28T00:00:00-07:00"> Tue 28 August 2001</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/solvation.html">solvation</a>
        /
	<a href="./tag/software.html">software</a>
        /
	<a href="./tag/apbs.html">apbs</a>
        /
	<a href="./tag/ribosome.html">ribosome</a>
        /
	<a href="./tag/microtubules.html">microtubules</a>
        /
	<a href="./tag/finite_differences.html">finite_differences</a>
        /
	<a href="./tag/electrostatics.html">electrostatics</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>Evaluation of the electrostatic properties of biomolecules has become a standard practice in molecular biophysics. Foremost among the models used to elucidate the electrostatic potential is the Poisson-Boltzmann equation; however, existing methods for solving this equation have limited the scope of accurate electrostatic calculations to relatively small biomolecular systems. Here we present the application of numerical methods to enable the trivially parallel solution of the Poisson-Boltzmann equation for supramolecular structures that are orders of magnitude larger in size. As a demonstration of this methodology, electrostatic potentials have been calculated for large microtubule and ribosome structures. The results point to the likely role of electrostatics in a variety of activities of these structures.</p>
                    <a class="btn btn-default btn-xs" href="./electrostatics-of-nanosystems-application-to-microtubules-and-the-ribosome.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./the-adaptive-multilevel-finite-element-solution-of-the-poisson-boltzmann-equation-on-massively-parallel-computers.html">The adaptive multilevel finite element solution of the Poisson-Boltzmann equation on massively parallel computers</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2001-01-01T00:00:00-08:00"> Mon 01 January 2001</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/solvation.html">solvation</a>
        /
	<a href="./tag/software.html">software</a>
        /
	<a href="./tag/microtubules.html">microtubules</a>
        /
	<a href="./tag/electrostatics.html">electrostatics</a>
        /
	<a href="./tag/finite_elements.html">finite_elements</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>By using new methods for the parallel solution of elliptic partial differential equations, the teraflops computing power of massively parallel computers can be leveraged to perform electrostatic calculations on large biological systems. This paper describes the adaptive multilevel finite element solution of the Poisson-Boltzmann equation for a microtubule on the NPACI Blue Horizon—a massively parallel IBM RS/6000® SP with eight POWER3 SMP nodes. The microtubule system is 40 nm in length and 24 nm in diameter, consists of roughly 600000 atoms, and has a net charge of −1800 e. Poisson-Boltzmann calculations are performed for several processor configurations, and the algorithm used shows excellent parallel scaling.</p>
                    <a class="btn btn-default btn-xs" href="./the-adaptive-multilevel-finite-element-solution-of-the-poisson-boltzmann-equation-on-massively-parallel-computers.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./adaptive-multilevel-finite-element-solution-of-the-poisson-boltzmann-equation-i-algorithms-and-examples.html">Adaptive multilevel finite element solution of the Poisson-Boltzmann equation I. Algorithms and examples</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2000-01-01T00:00:00-08:00"> Sat 01 January 2000</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/solvation.html">solvation</a>
        /
	<a href="./tag/software.html">software</a>
        /
	<a href="./tag/electrostatics.html">electrostatics</a>
        /
	<a href="./tag/finite_elements.html">finite_elements</a>
        /
	<a href="./tag/longmath.html">longmath</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>This article is the first of two articles On the adaptive multilevel finite element treatment of the nonlinear Poisson-Boltzmann equation (PBE), a nonlinear eliptic equation arising in biomolecular modeling. Fast and accurate numerical solution of the PBE is usually difficult to accomplish, due to the presence of discontinuous coefficients, delta functions, three spatial dimensions, unbounded domain, and rapid (exponential) nonlinearity. In this first article, we explain how adaptive multilevel finite element methods can be used to obtain extremely accurate solutions to the PBE with very modest computational resources, and we present some illustrative examples using two well-known test problems. The PBE is first discretized with piece-wise linear finite elements over a very coarse simplex triangulation of the domain. The resulting nonlinear algebraic equations are solved with global inexact Newton methods, which we have described in an article appearing previously in this journal. A posteriori error estimates are then computed from this discrete solution, which then drives a simplex subdivision algorithm for performing adaptive mesh refinement. The discretize-solve-estimate-refine procedure is then repeated, until a nearly uniform solution quality is obtained. The sequence of unstructured meshes is used to apply multilevel methods in conjunction with global inexact Newton methods, so that the cost of solving the nonlinear algebraic equations at each step approaches optimal O(N) linear complexity. All of the numerical procedures are implemented. in MANIFOLD CODE (MC), a computer program designed and built by the first author over several years at Caltech and UC San Diego. MC is designed to solve a very general class of nonlinear elliptic equations on complicated domains in two and three dimensions. We describe some of the key features of MC, and give a detailed analysis of its performance for two model PBE problems, with comparisons to the alternative methods. It is shown that the best available uniform mesh-based finite difference or box-method algorithms, including multilevel methods, require substantially more time to reach a target PBE solution accuracy than the adaptive multilevel methods in MC. in the second article, we develop an error estimator based on geometric solvent accessibility, and present a series of detailed numerical experiments for several complex biomolecules.</p>
                    <a class="btn btn-default btn-xs" href="./adaptive-multilevel-finite-element-solution-of-the-poisson-boltzmann-equation-i-algorithms-and-examples.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./adaptive-multilevel-finite-element-solution-of-the-poisson-boltzmann-equation-ii-refinement-at-solvent-accessible-surfaces-in-biomolecular-systems.html">Adaptive multilevel finite element solution of the Poisson-Boltzmann equation II. Refinement at solvent-accessible surfaces in biomolecular systems</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2000-01-01T00:00:00-08:00"> Sat 01 January 2000</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/solvation.html">solvation</a>
        /
	<a href="./tag/software.html">software</a>
        /
	<a href="./tag/electrostatics.html">electrostatics</a>
        /
	<a href="./tag/finite_elements.html">finite_elements</a>
        /
	<a href="./tag/longmath.html">longmath</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>We apply the adaptive multilevel finite element techniques (Holst, Baker, and Wang [21]) to the nonlinear Poisson-Boltzmann equation (PBE) in the context of biomolecules. Fast and accurate numerical solution of the PBE in this setting is usually difficult to accomplish due to presence of discontinuous coefficients, delta functions, three spatial dimensions, unbounded domains, and rapid (exponential) nonlinearity. However, these adaptive techniques have shown substantial improvement in solution time over conventional uniform-mesh finite difference methods. One important aspect of the adaptive multilevel finite element method is the robust a posteriori error estimators necessary to drive the adaptive refinement routines. This article discusses the choice of solvent accessibility for a posteriori error estimation of PBE solutions and the implementation of such routines in the Adaptive Poisson-Boltzmann Solver (APBS) software package based on the Manifold Code (MC) libraries. Results are shown for the application of this method to several biomolecular systems.</p>
                    <a class="btn btn-default btn-xs" href="./adaptive-multilevel-finite-element-solution-of-the-poisson-boltzmann-equation-ii-refinement-at-solvent-accessible-surfaces-in-biomolecular-systems.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./delineation-and-decomposition-of-energies-involved-in-quaternary-ammonium-binding-in-the-active-site-of-acetylcholinesterase.html">Delineation and decomposition of energies involved in quaternary ammonium binding in the active site of acetylcholinesterase</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2000-01-01T00:00:00-08:00"> Sat 01 January 2000</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/acetylcholinesterase.html">acetylcholinesterase</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>The quaternary ammonium binding locus in the active site of mammalian acetylcholinesterase is subtended by the side chains of Trp86, Tyr133, Glu202, and Tyr337. Linear free-energy relationships define the interactions involved in molecular recognition by mouse acetylcholinesterase of the quaternary ammonium moiety of ligands. For substrates CH3C(O)XCH2CH2Y [X = O, Y = CHMe2, or CH2CH3; X = S, Y = H, NH+Me2, or N+Me3 ] and trifluoroacetophenone transition state analogue inhibitors m-YC6H4C(O)CF3 [Y = H, Me, Et, iPr, tBu, CF3, NH2, NO2, NMe2, or N+Me3], log(kcat/Km) and pKi depend linearly on the molar refractivity, but not the hydrophobicity, of the substituents Y. These correlations indicate that, in the acylation stage of catalysis, interactions in the quaternary ammonium binding locus stabilize the tetrahedral intermediate (as modeled by transition state analogue affinity) by (5 105)-fold (GTI = 32.5 kJ mol-1) and the transition state by (2 104)-fold (G = 24.5 kJ mol-1). To evaluate the contribution of cation- interactions, Trp86 was converted into Tyr, Phe, and Ala by site-specific mutagenesis. For this set of enzymes, a linear free-energy relationship is observed between the pKi values for inhibitions by the respective neutral and cationic transition state analogue inhibitors, m-tert-butyltrifluoroacetophenone and m-(N,N,N-trimethylammonio)trifluoroacetophenone, which indicates that the free energy released on interaction of the quaternary ammonium moiety with Trp86 arises about equally from cation- and charge-independent interactions. The quaternary ammonium binding locus in the active site of mammalian acetylcholinesterase is subtended by the side chains of Trp86, Tyr133, Glu202, and Tyr337. Linear free-energy relationships define the interactions involved in molecular recognition by mouse acetylcholinesterase of the quaternary ammonium moiety of ligands. For substrates CH3C(O)XCH2CH2Y [X = O, Y = CHMe2, or CH2CH3; X = S, Y = H, NH+Me2, or N+Me3 ] and trifluoroacetophenone transition state analogue inhibitors m-YC6H4C(O)CF3 [Y = H, Me, Et, iPr, tBu, CF3, NH2, NO2, NMe2, or N+Me3], log(kcat/Km) and pKi depend linearly on the molar refractivity, but not the hydrophobicity, of the substituents Y. These correlations indicate that, in the acylation stage of catalysis, interactions in the quaternary ammonium binding locus stabilize the tetrahedral intermediate (as modeled by transition state analogue affinity) by (5 105)-fold (GTI = 32.5 kJ mol-1) and the transition state by (2 104)-fold (G = 24.5 kJ mol-1). To evaluate the contribution of cation- interactions, Trp86 was converted into Tyr, Phe, and Ala by site-specific mutagenesis. For this set of enzymes, a linear free-energy relationship is observed between the pKi values for inhibitions by the respective neutral and cationic transition state analogue inhibitors, m-tert-butyltrifluoroacetophenone and m-(N,N,N-trimethylammonio)trifluoroacetophenone, which indicates that the free energy released on interaction of the quaternary ammonium moiety with Trp86 arises about equally from cation- and charge-independent interactions.</p>
                    <a class="btn btn-default btn-xs" href="./delineation-and-decomposition-of-energies-involved-in-quaternary-ammonium-binding-in-the-active-site-of-acetylcholinesterase.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./polarization-around-an-ion-in-a-dielectric-continuum-with-truncated-electrostatic-interactions.html">Polarization around an ion in a dielectric continuum with truncated electrostatic interactions</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="1999-01-01T00:00:00-08:00"> Fri 01 January 1999</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/solvation.html">solvation</a>
        /
	<a href="./tag/electrostatics.html">electrostatics</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>In order to reduce computational effort and to allow for the use of periodic boundary conditions,electrostatic interactions in explicit solvent simulations of molecular systems do not obey Coulomb’s law. Instead, a number of “effective potentials” have been proposed, including truncated Coulomb, shifted, switched, reaction-field corrected, or Ewald potentials. The present study compares the performance of these schemes in the context of ionic solvation. To this purpose, a generalized form of the Born continuum model for ion solvation is developed, where ion–solvent and solvent–solvent interactions are determined by these effective potentials instead of Coulomb’s law. An integral equation is formulated for calculating the polarization around a spherical ion from which the solvation free energy can be extracted. Comparison of the polarizations and free energies calculated for specific effective potentials and the exact Born result permits an assessment of the accuracy of these different schemes. Additionally, the present formalism can be used to develop corrections to the ionic solvation free energies calculated by molecular simulations implementing such effective potentials. Finally, an arbitrary effective potential is optimized to reproduce the Born polarization.</p>
                    <a class="btn btn-default btn-xs" href="./polarization-around-an-ion-in-a-dielectric-continuum-with-truncated-electrostatic-interactions.html">more ...</a>
                </div>
            </article>
            <hr/>
            <article>
                <h2><a href="./molecular-recognition-by-cholesterol-esterase-of-active-site-ligands-structure-reactivity-effects-for-inhibition-by-aryl-carbamates-and-subsequent-carbamylenzyme-turnover.html">Molecular recognition by cholesterol esterase of active site ligands: structure-reactivity effects for inhibition by aryl carbamates and subsequent carbamylenzyme turnover</a></h2>
                    <div class="well well-sm">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="1996-01-01T00:00:00-08:00"> Mon 01 January 1996</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="./tag/cholesterol.html">cholesterol</a>
    
</footer><!-- /.post-info -->                    </div>
                <div class="summary"><p>Interactions of mammalian pancreatic cholesterol esterases from pig and rat with a family of aryl carbamates CnH2n+1NHCOOAr [n = 4-9; Ar = phenyl, p-X-phenyl (X = acetamido, bromo, fluoro, nitro, trifluoromethyl), 2-naphthyl, 2-tetrahydronaphthyl, estronyl] have been investigated, with an aim of delineating the ligand structural features which lead to effective molecular recognition by the active site of the enzyme. These carbamates inhibit the catalytic activity of CEase by rapid carbamylation of the active site, a process that shows saturation kinetics. Subsequent slow decarbamylation usually leads to full restoration of activity, and therefore aryl carbamates are transient inhibitors, or pseudo-substrates, of CEase. Structural variation of carbamate inhibitors allowed molecular recognition in the fatty acid binding and steroid binding loci of the extended active site to be probed, and the electronic nature of the carbamylation transition state to be characterized. Optimal inhibitory activity is observed when the length of the carbamyl function is n = 6 and n = 7 for porcine and rat cholesterol esterases, respectively, equivalent to eight- and nine-carbon fatty acyl chains. In contrast, inhibitory activity increases progressively as the partial molecular volume of the aromatic fragment increases. Hammett plots for p-substituted phenyl-N-hexyl carbamates indicate that the rate-determining step for carbamate inhibition is phenolate anion expulsion. Effects of the bile salt activator taurocholate on the kinetically resolved phases of the pseudo-substrate turnover of aryl carbamates were also studied. Taurocholate increases the affinity of the carbamate for the active site of cholesterol esterase in the reversible, noncovalent complex that precedes carbamylation and increases the rate constants of the serial carbamylation and decarbamylation steps. Structural variation of the N-alkyl chain and of the aryl fused-ring system provides an accounting of bile salt modulation of the fatty acid and steroid binding sites, respectively. In that pseudo-substrate turnover of aryl carbamates proceeds by a three-step mechanism that is analogous to that for rapid turnover of lipid ester substrates, these investigations illuminate details of ligand recognition by the extended active site of cholesterol esterase that are prominent determinants of the substrate specificity and catalytic power of the enzyme.</p>
                    <a class="btn btn-default btn-xs" href="./molecular-recognition-by-cholesterol-esterase-of-active-site-ligands-structure-reactivity-effects-for-inhibition-by-aryl-carbamates-and-subsequent-carbamylenzyme-turnover.html">more ...</a>
                </div>
            </article>
            <hr/>

        </div>
        <div class="col-sm-3" id="sidebar">
            <aside>

<section class="well well-sm">
    <ul class="list-group list-group-flush">
            <li class="list-group-item"><h4><i class="fa fa-home fa-lg"></i><span class="icon-label">Social</span></h4>
              <ul class="list-group" id="social">
                <li class="list-group-item"><a href="http://www.linkedin.com/in/nathanandrewbaker"><i class="fa fa-linkedin-square fa-lg"></i> LinkedIn</a></li>
                <li class="list-group-item"><a href="https://www.mendeley.com/profiles/nathan-baker1/"><i class="fa fa-mendeley-square fa-lg"></i> Mendeley</a></li>
              </ul>
            </li>



            <li class="list-group-item"><a href="./"><h4><i class="fa fa-tags fa-lg"></i><span class="icon-label">Tags</span></h4></a>
                <ul class="list-group list-inline tagcloud" id="tags">
                    <li class="list-group-item tag-2">
                        <a href="./tag/acetylcholinesterase.html">
                            acetylcholinesterase
                        </a>
                    </li>
                    <li class="list-group-item tag-4">
                        <a href="./tag/allostery.html">
                            allostery
                        </a>
                    </li>
                    <li class="list-group-item tag-4">
                        <a href="./tag/apbs.html">
                            apbs
                        </a>
                    </li>
                    <li class="list-group-item tag-2">
                        <a href="./tag/biomembranes.html">
                            biomembranes
                        </a>
                    </li>
                    <li class="list-group-item tag-3">
                        <a href="./tag/cholesterol.html">
                            cholesterol
                        </a>
                    </li>
                    <li class="list-group-item tag-4">
                        <a href="./tag/coarse-graining.html">
                            coarse-graining
                        </a>
                    </li>
                    <li class="list-group-item tag-2">
                        <a href="./tag/current_research.html">
                            current_research
                        </a>
                    </li>
                    <li class="list-group-item tag-2">
                        <a href="./tag/diffusion.html">
                            diffusion
                        </a>
                    </li>
                    <li class="list-group-item tag-3">
                        <a href="./tag/dna.html">
                            dna
                        </a>
                    </li>
                    <li class="list-group-item tag-1">
                        <a href="./tag/electrostatics.html">
                            electrostatics
                        </a>
                    </li>
                    <li class="list-group-item tag-3">
                        <a href="./tag/finite_differences.html">
                            finite_differences
                        </a>
                    </li>
                    <li class="list-group-item tag-1">
                        <a href="./tag/finite_elements.html">
                            finite_elements
                        </a>
                    </li>
                    <li class="list-group-item tag-4">
                        <a href="./tag/force_fields.html">
                            force_fields
                        </a>
                    </li>
                    <li class="list-group-item tag-4">
                        <a href="./tag/geometric_flow.html">
                            geometric_flow
                        </a>
                    </li>
                    <li class="list-group-item tag-2">
                        <a href="./tag/informatics.html">
                            informatics
                        </a>
                    </li>
                    <li class="list-group-item tag-4">
                        <a href="./tag/ion_channels.html">
                            ion_channels
                        </a>
                    </li>
                    <li class="list-group-item tag-3">
                        <a href="./tag/longmath.html">
                            longmath
                        </a>
                    </li>
                    <li class="list-group-item tag-4">
                        <a href="./tag/machine_learning.html">
                            machine_learning
                        </a>
                    </li>
                    <li class="list-group-item tag-4">
                        <a href="./tag/materials.html">
                            materials
                        </a>
                    </li>
                    <li class="list-group-item tag-2">
                        <a href="./tag/mathematics.html">
                            mathematics
                        </a>
                    </li>
                    <li class="list-group-item tag-3">
                        <a href="./tag/metalloregulation.html">
                            metalloregulation
                        </a>
                    </li>
                    <li class="list-group-item tag-2">
                        <a href="./tag/microtubules.html">
                            microtubules
                        </a>
                    </li>
                    <li class="list-group-item tag-2">
                        <a href="./tag/molecular_dynamics.html">
                            molecular_dynamics
                        </a>
                    </li>
                    <li class="list-group-item tag-3">
                        <a href="./tag/monte_carlo.html">
                            monte_carlo
                        </a>
                    </li>
                    <li class="list-group-item tag-3">
                        <a href="./tag/multiscale.html">
                            multiscale
                        </a>
                    </li>
                    <li class="list-group-item tag-2">
                        <a href="./tag/nanotechnology.html">
                            nanotechnology
                        </a>
                    </li>
                    <li class="list-group-item tag-3">
                        <a href="./tag/nikr.html">
                            nikr
                        </a>
                    </li>
                    <li class="list-group-item tag-3">
                        <a href="./tag/nonpolar.html">
                            nonpolar
                        </a>
                    </li>
                    <li class="list-group-item tag-3">
                        <a href="./tag/nucleic_acids.html">
                            nucleic_acids
                        </a>
                    </li>
                    <li class="list-group-item tag-2">
                        <a href="./tag/ontology.html">
                            ontology
                        </a>
                    </li>
                    <li class="list-group-item tag-4">
                        <a href="./tag/oxysterols.html">
                            oxysterols
                        </a>
                    </li>
                    <li class="list-group-item tag-1">
                        <a href="./tag/past_research.html">
                            past_research
                        </a>
                    </li>
                    <li class="list-group-item tag-3">
                        <a href="./tag/review.html">
                            review
                        </a>
                    </li>
                    <li class="list-group-item tag-3">
                        <a href="./tag/ribosome.html">
                            ribosome
                        </a>
                    </li>
                    <li class="list-group-item tag-3">
                        <a href="./tag/rna.html">
                            rna
                        </a>
                    </li>
                    <li class="list-group-item tag-4">
                        <a href="./tag/salicylate.html">
                            salicylate
                        </a>
                    </li>
                    <li class="list-group-item tag-4">
                        <a href="./tag/signatures.html">
                            signatures
                        </a>
                    </li>
                    <li class="list-group-item tag-1">
                        <a href="./tag/software.html">
                            software
                        </a>
                    </li>
                    <li class="list-group-item tag-1">
                        <a href="./tag/solvation.html">
                            solvation
                        </a>
                    </li>
                    <li class="list-group-item tag-4">
                        <a href="./tag/statistics.html">
                            statistics
                        </a>
                    </li>
                    <li class="list-group-item tag-2">
                        <a href="./tag/titration.html">
                            titration
                        </a>
                    </li>
                    <li class="list-group-item tag-4">
                        <a href="./tag/uq.html">
                            uq
                        </a>
                    </li>
                    <li class="list-group-item tag-3">
                        <a href="./tag/virus.html">
                            virus
                        </a>
                    </li>
                    <li class="list-group-item tag-3">
                        <a href="./tag/working_group.html">
                            working_group
                        </a>
                    </li>
                </ul>
            </li>


    <li class="list-group-item"><h4><i class="fa fa-external-link-square fa-lg"></i><span class="icon-label">Links</span></h4>
      <ul class="list-group" id="links">
        <li class="list-group-item">
            <a href="https://scholar.google.com/citations?user=L9dwKyUAAAAJ&hl=en" target="_blank">
                Google Scholar
            </a>
        </li>
        <li class="list-group-item">
            <a href="http://orcid.org/0000-0002-5892-6506" target="_blank">
                ORCID
            </a>
        </li>
      </ul>
    </li>
    </ul>
</section>
            </aside>
        </div>
    </div>
</div>
<footer>
   <div class="container">
      <hr>
      <div class="row">
         <div class="col-xs-10">&copy; 2012 Nathan Baker
            &middot; Powered by <a href="https://github.com/DandyDev/pelican-bootstrap3" target="_blank">pelican-bootstrap3</a>,
            <a href="http://docs.getpelican.com/" target="_blank">Pelican</a>,
            <a href="http://getbootstrap.com" target="_blank">Bootstrap</a>         </div>
         <div class="col-xs-2"><p class="pull-right"><i class="fa fa-arrow-up"></i> <a href="#">Back to top</a></p></div>
      </div>
   </div>
</footer>
<script src="./theme/js/jquery.min.js"></script>

<!-- Include all compiled plugins (below), or include individual files as needed -->
<script src="./theme/js/bootstrap.min.js"></script>

<!-- Enable responsive features in IE8 with Respond.js (https://github.com/scottjehl/Respond) -->
<script src="./theme/js/respond.min.js"></script>

    <!-- Google Analytics -->
    <script type="text/javascript">

        var _gaq = _gaq || [];
        _gaq.push(['_setAccount', 'UA-11026338-13']);
        _gaq.push(['_trackPageview']);

        (function () {
            var ga = document.createElement('script');
            ga.type = 'text/javascript';
            ga.async = true;
            ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
            var s = document.getElementsByTagName('script')[0];
            s.parentNode.insertBefore(ga, s);
        })();
    </script>
    <!-- End Google Analytics Code -->

</body>
</html>